frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-page/common-page.component.ts

```diff
@@ -5,15 +5,32 @@ import { TranslateModule } from '@ngx-translate/core';
 /**
  * A component nested in this component can be displayed in the **header** or **content** slot.
  * The **header** slot should contain a component that wraps a **app-common-page-header**.
- * The **content** slot should contain a component that wraps a list of **app-common-section-card**.
+ * The **section-card** slot should contain section-card components.
  * Simply nest the component and add the slot name as an attribute.
  *
  * Example usage:
  *
  * ```
  * <app-common-page>
- *   <div header>Hi I'm in the header slot</div>
- *   <div content>Hi I'm in the content slot</div>
+ *   <div header>header</div>
+ *   <div section-card>section-card 1</div>
```

Comment:
Do you think this is the right way? I feel common page HTML shouldn't be affected by this change

Reply:
there didn't seem to be anything that pulling out this structure into a special `section card list` would offer if this is the only kind of "content" that a page should ever expect. Later if we find that's not the case, where a page may not use section-cards, we can pretty easily adapt that and pull this structure into its own component. That was the reasoning.